development/compare-vol.py

```diff
@@ -339,7 +339,7 @@ def run_test(self, plugin: VolatilityPlugin, image: VolatilityImage):
         "--vol3path",
         type=str,
         default=os.path.join(os.getcwd(), "volatility3"),
-        help="Path ot the volatility 3 directory",
+        help="Path of the volatility 3 directory",
```

Comment:
```suggestion
        help="Path to the volatility 3 directory",
```
I think this was originally supposed to say path to the directory, rather than of.